### PR TITLE
Add net.sf.fuse_emulator

### PR DESCRIPTION
**Pull request by bertogg**
I'm submitting a new app to Flathub.

This is Fuse, an emulator for the Sinclair ZX Spectrum home computer. I think the packaging itself is quite simple and the app has very few dependencies, but this if my first submission to Flathub so I expect that there could be mistakes or things to improve.

I was initially going to use the GNOME runtime but Fuse only depends on GTK+ 3, which is already available in Freedesktop, so I chose the latter in the end.

The code is GPLv2, except for the Spectrum ROMs (these are not games, it's the firmware of the computer). These ROMs have been legally distributable for almost 20 years now. The copyright holder (Amstrad) has given permission to distribute them freely for use with emulators as long as no one charges for them. See https://sourceforge.net/p/fuse-emulator/fuse/ci/master/tree/roms/README.copyright for more details.

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build net.sf.fuse_emulator

File: net.sf.fuse_emulator.json [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+                    "path": "audiofile-build.patch"
+                }
+            ],
+            "cleanup": [
+                "/bin"
+            ]
+        },
+        {
+            /* SDL is needed for joystick support */
+            "name": "libsdl",
```
Comment (barthalion) [inline net.sf.fuse_emulator.json]: Please use the one from `shared-modules` (as git submodule).

**Comment by bertogg** [comment]
> Have you contacted upstream if they are interested in co-maintaining the app? It would be also good to contribute appdata there.

I work with the upstream team, so yes, everyone is aware that I'm doing this. I think adding the appdata for a future release is not going to be a problem either.

File: net.sf.fuse_emulator.json [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                    "type": "patch",
+                    "path": "audiofile-build.patch"
+                }
+            ],
+            "cleanup": [
+                "/bin"
+            ]
+        },
+        "shared-modules/SDL/SDL-1.2.15.json",
+        {
```
Comment (barthalion) [inline net.sf.fuse_emulator.json]: Some leftovers from removal?

Comment (bertogg) [reply net.sf.fuse_emulator.json]: What leftovers? I'm including the SDL json there :-?

Comment (bertogg) [reply net.sf.fuse_emulator.json]: Oh, I see the "name": "libsdl" part is not necessary.

I was confused by the instructions here:

https://github.com/flathub/flathub/wiki/App-Requirements#shared-modules

**Comment by barthalion** [comment]
bot, build net.sf.fuse_emulator

**Comment by barthalion** [comment]
https://flathub.org/builds/#/builders/2/builds/2250 another build succeeded

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/net.sf.fuse_emulator

Make sure to accept the invitation. Should someone from upstream team also have access to it?

Thanks!

**Comment by bertogg** [comment]
> Make sure to accept the invitation. Should someone from upstream team also have access to it?

Thanks! I'm from upstream myself, but I'll indeed give access to other members who are interested in helping.

**Comment by barthalion** [comment]
Yeah, that's what I meant. Just ping me on IRC or file an issue on Flathub bug tracker, you need someone with admin privileges for that.

**Pull request closed without merging**